lua/xcodebuild/integrations/xcode_build_server.lua

```diff
@@ -0,0 +1,15 @@
+---@mod xcodebuild.integrations.xcode_build_server xcode-build-server Integration
+---@brief [[
+---This module is responsible for the integration with xcode-build-server.
+---It provides a function to update buildServer.json file with the current scheme.
+---@brief ]]
+
+local M = {}
+
+---Calls "config" command of xcode-build-server in order to update buildServer.json file.
+---@param scheme string
+function M.run_config(projectCommand, scheme)
+  vim.fn.jobstart("xcode-build-server config " .. projectCommand .. " -scheme '" .. scheme .. "'")
+end
```

Comment:
Please add also annotation for the second param and I think it would be nice here to return job id in case it's needed in the future (just add `return`) and `---@return number # job id`.

I forgot to mention in the guide. If you add a new module you need to list it also in `scripts/update-help.sh` so that it can appear in the help.